NEWS.md

```diff
@@ -1,5 +1,13 @@
 # History of user-visible changes
 
+## 2019-06-03
```

Comment:
This should say `## Next` for now. These are dates of actual releases.

Reply:
Done.